app/konnect/deployment/import.md

```diff
@@ -0,0 +1,136 @@
+
+    ```sh
+    deck ping \
+      --konnect-runtime-group default \
+      --konnect-email {YOUR_EMAIL} \
+      --konnect-password {YOUR_PASSWORD}
+    ```
+
+    You can also use decK with {{site.konnect_short_name}} more securely by storing
+    your password in a file, then either calling it with
+    `--konnect-password-file {FILENAME}.txt`, or adding it to your decK configuration
+    under the `konnect-password` option along with your email:
+
+    ```yaml
+    konnect-password: {YOUR_PASSWORD}
+    konnect-email: {YOUR_EMAIL}
+    ```
+
+    The following steps all use a `deck.yaml` file to store the
+    {{site.konnect_short_name}} credentials instead of flags.
+
+1. Export configuration from {{site.base_gateway}} with [`deck dump`](/deck/latest/reference/deck_dump):
+
+    ```bash
+    deck dump --output-file konnect.yaml
+    ```
+
+    This command outputs {{site.base_gateway}}'s object configuration into the
+    specified file.
+
+1. Preview the import with the [`deck diff`](/deck/latest/reference/deck_diff)
+command, pointing to the runtime group that you want to target:
+
+    ```sh
+    deck diff \
+      --state konnect.yaml \
+      --konnect-runtime-group default
+    ```
+
+1. If you're satisfied with the preview, run [`deck sync`](/deck/latest/reference/deck_sync):
+
+    ```sh
+    deck sync \
+      --state konnect.yaml \
+      --konnect-runtime-group default
+    ```
+
+    If you don't specify the `--konnect-runtime-group` flag, decK targets the
+    `default` runtime group. If you have more than one runtime group in your
+    organization, we recommend always setting this flag to avoid accidentally
+    pushing configuration to the wrong group.
+
+1. Log in to your [{{site.konnect_saas}}](http://cloud.konghq.com/login) account.
+
+1. From the left navigation menu, open **Runtimes**, then open the runtime group
+you just updated.
+
+1. Look through the configuration details of any imported entities to make sure
+they were migrated successfully.
+
+## Migrate data planes
+
+You can keep any data plane nodes that are:
+* Running {{site.base_gateway}} (not the open-source package)
+* Are at least version 2.3 or higher
+
+Turn any self-managed nodes into cloud data plane nodes by registering them
+through the Runtime Manager and adjusting their configurations, or power down
+the old instances and create new data plane nodes through {{site.konnect_saas}}.
+
+1. Follow the [runtime setup guide](/konnect/runtime-manager/#kong-gateway) for
+your preferred deployment type.
+
+2. Once you have created or converted the data plane nodes, `kong stop` your
+old Gateway runtimes, then shut them down.
+
+3. If any of the old nodes have connected PostgreSQL or Cassandra instances,
+you can shut them down now.
+
+## Post-migration tasks
+
+See the following docs to set up any additional things you may need:
+
+* **Dev Portal files:** You can migrate API specs and markdown documents using the
```

Comment:
This feels misleading as there is not parity between portals. Is it worth calling out that Konnect only supports a single markdown service document to manage expectations?